chef/cookbooks/metasploitable/attributes/default.rb

```diff
@@ -18,5 +18,6 @@
                                      :chatbot_nodejs => 3000,
                                      :readme_app => 3500,
                                      :sinatra => 8181,
-                                     :samba => 445
+                                     :samba => 445,
+                                     :ssh => 22
```

Comment:
I would suggest that `ssh` should not be added here.  The array here is `default[:metasploitable][:ports]` the intent of this variable is to be a list of ports that support vulnerable paths on the VM you might target with `metasploit-framework`.  Retaining the separation of `ssh` in the source is intended to make it clear the port is being allowed explicitly without a vuln attached to it.

Reply:
In that case, how do you feel about putting the port-opening rules contained within each vuln's recipe? In the name of modularization -- e.g., if the `readme_app` recipe isn't included, then its rule shouldn't be added to iptables, no?

Reply:
I like the idea of extracting the rules into the recipes they are connected to.

Reply:
Done